### ISSUE TITLE
some new idea

### ISSUE THREAD

**Issue by justingranger12** [EastItem/auto-daka#2]
First of all, I would like to express my gratitude to you for sharing the project. Thank you for your selfless dedication and the open source spirit of github.
！@For feasibility, I have implemented the following operations, so don't worry
 //For the program, I think you can add a [random] library to let the program sleep for a random time after it is triggered, because it may be a little strange if you check in super on time every day. 
// The second point is that I think some feedback mechanisms can be added. Here I recommend the [虾推啥] WeChat public account. After you follow it, you will naturally understand how to perform feedback operations. 
That's all, allow me to thank you again for the program you shared.

**Comment by EastItem** [comment]
感谢你的反馈！对于你的两点反馈，我有如下说明。
第一，关于取随机时间进行打卡。这里建议更改触发器的设定。如果在代码里增加随机延迟的话，延迟短了没什么用，延迟长了浪费云函数的资源使用量。
第二，关于打卡通知设定，之前是有考虑过用“server酱”进行通知，但由于最近比较忙，所以搁置了。在下次更新中，会考虑添加。
综上，在下次更新中，会添加如何随机时间打卡的说明，以及添加打卡反馈。感谢你的反馈！

Thank you for your feedback! I have the following explanation for your two points of feedback.

First, about taking random time to check in. It is recommended to change the settings of the trigger. If random latency is added to the code, it is useless to short the delay, and the delay will waste the resource usage of cloud functions.

Second, Regarding The Setting Of Punching In Notification, I Have Considered Using "server 酱" To Make Notifications Before, But Because I Have Been Busy Recently, It Has Been Shelved. In the next update, you will consider adding it.

In summary, in the next update, instructions on how to punch in at random times and punch feedback will be added. Thank you for your feedback!

**Comment by EastItem** [comment]
已更新！

**Comment by EastItem** [comment]
看了一下，你推荐的[虾推啥] ，感觉确实好用一点。会在下次更新中调整平台，谢谢推荐！
